### ISSUE TITLE
Is oneops safe to deploy in a cloud with existing infrastructure?

### ISSUE THREAD

**Issue by empath** [oneops/OneOps#7]
I deployed it on a test account and followed the deploy with the test assembly and I noticed in the logs that it was running queries on currently running instances.  

Why is it doing that, and will it modify existing infrastructure that isn't configured somewhere in oneops?

**Comment by kfilipovski** [comment]
@empath Any resources created in the cloud(s) by OneOps will usually have the unique OneOps ID (ciId) as part of the name, so there should be no conflicts with any non-OneOps managed resources under the same cloud account.  It's possible that some component recipes will perform a 'list' call in order to check if the expected resources are there or not in which case you will see the non-managed stuff in the logs. If you can track down the place in the code in https://github.com/oneops/circuit-oneops-1 based on your logs (or join our slack team and I can help track it down) we can take a look at either modifying the query to avoid this or maybe just suppress the extra logging

Bottom line, it should be safe to use in parallel with existing infrastructure, but it's worth noting couple of things:
- OneOps has been open-source for few weeks only, so not much time for community to discover any issues related to this area
- OneOps has been used in production at Walmart for some time now, but there is very little cloud use NOT managed by oneops, meaning not much opportunity to discover issues related to conflicts with existing infrastructure

Obviously, creating separate identity/credentials/tenants, if possible, during an evaluation period would be preferred.
